projects/cloud/app/frontend/stores/HomeStore.ts

```diff
@@ -0,0 +1,25 @@
+import { makeAutoObservable } from 'mobx';
+import OrganizationStore from './OrganizationStore';
+import UserStore from './UserStore';
+import { createContext } from 'react';
+import { ApolloClient } from '@apollo/client';
+
+export class HomeStore {
+  userStore: UserStore;
+  organizationStore: OrganizationStore;
+  client: ApolloClient<object>;
+
+  constructor(client: ApolloClient<object>) {
+    makeAutoObservable(this);
+    this.userStore = new UserStore(client);
+    this.organizationStore = new OrganizationStore(client);
+  }
+
+  async load(organizationName: string) {
+    await this.userStore.load();
+    await this.organizationStore.load(organizationName);
+  }
+}
+
+// @ts-ignore
+export const HomeStoreContext = createContext<HomeStore>();
```

Comment:
Have you considered using [this integration](https://mobx.js.org/react-integration.html) instead?

Reply:
what exactly do you mean here? I am using `observer` for components but I did not want to pass the store props all over the place as context is more convenient.